.github/workflows/merge-main.yml

```diff
@@ -129,6 +129,7 @@ jobs:
             -p REACT_APP_KC_REALM=standard
             -p REACT_APP_KC_CLIENT_ID=seed-planning-test-4296
             -p REACT_APP_ENABLE_MOCK_SERVER=true
+            -p MIN_REPLICAS=1 -p MAX_REPLICAS=2
```

Comment:
This line was removed to match the removal from PROD. Typically those pick up the default from the template. If I got that wrong, please change PROD as well!